src/events/embedded/events.ts

```diff
@@ -1,63 +1,102 @@
 import { baseIterableRequest } from '../../request';
 import {
-  EmbeddedMessagingDismiss,
-  EmbeddedMessagingSession,
-  IEmbeddedMessageData,
-  IEmbeddedSession
-} from '../../../src/events/embedded/types';
+  EmbeddedDismissRequestPayload,
+  EmbeddedSessionRequestPayload,
+  EmbeddedClickRequestPayload
+} from './types';
 import { IterableResponse } from '../../types';
 import {
-  trackEmbeddedMessageSchema,
-  trackEmbeddedSessionSchema,
-  embaddedMessagingDismissSchema,
-  embaddedMessagingSessionSchema
+  trackEmbeddedSchema,
+  trackEmbeddedClickSchema,
+  embeddedDismissSchema,
+  embeddedSessionSchema
 } from './events.schema';
 import { EndPoints } from '../consts';
+import { WEB_PLATFORM } from 'src/constants';
 
-export const trackEmbeddedMessageReceived = (payload: IEmbeddedMessageData) => {
-  return baseIterableRequest<IterableResponse>({
+export const trackEmbeddedReceived = (
+  messageId: string,
+  appPackageName?: string
+) =>
+  baseIterableRequest<IterableResponse>({
     method: 'POST',
     url: EndPoints.msg_received_event_track,
-    data: payload,
+    data: {
+      messageId,
+      deviceInfo: {
+        platform: WEB_PLATFORM,
+        deviceId: global.navigator.userAgent || '',
+        appPackageName: appPackageName || window?.location?.hostname
```

Comment:
Curious, what is the hostname referring to that it should be assigned to appPackageName?